modules/core/src/main/java/org/apache/synapse/core/axis2/SynapseCallbackReceiver.java

```diff
@@ -201,12 +208,7 @@ public void receive(MessageContext messageCtx) throws AxisFault {
                 }
             } else {
                 // TODO invoke a generic synapse error handler for this message
-                if (!PassThroughConstants.INTERNAL_ORIGIN_ERROR_HANDLER
-                        .equals(messageCtx.getProperty(PassThroughConstants.INTERNAL_EXCEPTION_ORIGIN))) {
-                    log.warn("Synapse received a response for the request with message Id : " + messageID
-                            + " and correlation_id : " + messageCtx.getProperty(CorrelationConstants.CORRELATION_ID)
-                            + " But a callback is not registered (anymore) to process " + "this response");
-                }
+                logWarnNoCallbackRegistered(messageID, messageCtx);
```

Comment:
can we please rename this method name to something like "handleNoCallbackRegistered"

Reply:
renamed to handleNoCallback